docs/release-notes/2.24.0.md

```diff
@@ -19,6 +19,19 @@
 </table>
 
 Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+## Release Highlights
+- **Better PostgreSQL monitoring**
+  - Starting from this version of PMM, it will support monitoring of PostgreSQL 14. Both versions - Community and Percona Distribution for PostgreSQL.
```

Comment:
```suggestion
  - Starting from this version of PMM, it will support monitoring of PostgreSQL 14. Both versions - Community and [Percona Distribution for PostgreSQL](https://www.percona.com/software/postgresql-distribution).
```